functions/src/stories.ts

```diff
@@ -28,22 +34,45 @@ type StoryRecord = {
 type MetaData = {
   authorID: string,
   contributorsIDs?: string[],
-  lastUpdateAt?: firestore.Timestamp,
+  lastUpdateAt?: Timestamp,
   imageUrl?:string,
   likes?: number,
   views?: number,
   lastIndexedViews?: number,
   status?: "public" | "unlisted" | "private" | "pending"
 };
 
+type StoryDoc = {
+  actors: Array<{avatarUrl: string; id: string}>;
```

Comment:
question: why do we need actors (especially while omitting nodes)?

Reply:
We don't need. I was adding types for all fields, then realized that the nodes (and other fields) are too big and irrelevant for now, so I thought I'd add them when needed